tests/static_map/fetch_insert_test.cu

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2022, Jonas Hahnfeld, CERN.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <utils.hpp>
+
+#include <cuco/static_map.cuh>
+
+#include <thrust/device_vector.h>
+#include <thrust/functional.h>
+
+#include <catch2/catch.hpp>
+
+static constexpr int Iters = 10'000;
+
+template <typename Key, typename Value>
+__global__ void parallel_sum(typename cuco::static_map<Key, Value>::device_mutable_view m)
```

Comment:
I would just use an additional template param `typename FilterView` and let the compiler figure out the correct type.